Pinta/Dialogs/DocumentPropertiesDialog.cs

```diff
@@ -0,0 +1,202 @@
+// 
+// DocumentPropertiesDialog.cs
+//
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Gtk;
+using Mono.Unix;
+using Pinta.Core;
+
+namespace Pinta
+{
+    public class DocumentPropertiesDialog : Dialog
+    {
+        private DocumentProperties initial_properties;
+
+        public string author;
+        public string title;
+        public string subject;
+        public string keywords;
+        public string comments;
+
+        private Entry authorEntry;
+        private Entry titleEntry;
+        private Entry subjectEntry;
+        private Entry keywordsEntry;
+        private TextView commentsTextView; 
+        
+        public DocumentPropertiesDialog () : base (Catalog.GetString ("Properties"), PintaCore.Chrome.MainWindow, DialogFlags.Modal, Stock.Cancel, ResponseType.Cancel, Stock.Ok, ResponseType.Ok)
+        {
+            Build ();
+
+            this.Icon = PintaCore.Resources.GetIcon (Stock.Properties, 16);
+            
+            // suggest current user as author name
```

Comment:
Basically yes. 
It more than a little creepy having these filled in automatically. 
I had a go at suggestion completion but it was an ugly mess and deleted it, but it did seem like a less invasive way to give the user some helpful info.